src/_locales/ar/messages.json

```
@@ -0,0 +1,429 @@
+    },
+  
+    "optionDebugMode": {
+      "message": "تمكين وضع التفحيص",
+      "description": "This is an option shown in the add-on settings."
+    },
+    "optionDebugModeDescr": {
+      "message": "هذا مفيد فقط للحصول على ناتج تسجيل أكثر تفصيلاً في وحدة التحكم للإبلاغ عن الأخطاء ، إلخ",
+      "description": "The description shown for the debug mode (optionDebugMode)."
+    },
+  
+    "translatorCredit": {
+      "message": "This add-on has been translated into Arabic by Ahmad Ayman ($LINK$)!",
+      "description": "The credit text for the translator. See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on for how to translate this.",
+      "placeholders": {
+        "translators": {
+          "content": "$1",
+          "example": "<a href=\"https://github.com/AhmadAymanA99\">@AhmadAymanA99</a>"
+        }
+      }
+    },
+    "translatorLink": {
+      "message": "https://github.com/AhmadAymanA99",
+      "description": "The link to the translator's GitHub profile."
+    },
+    "translatorUsername": {
+      "message": "@AhmadAymanA99",
+      "description": "The username that the translator wants to be referred to."
+    },
+  
+    "contributorsThanks": {
+      "message": "شكرا لجميع المشاركين $CONTRIBUTORS$.",
+      "description": "Text thanking all contributors and linking to the contributors file.",
+      "placeholders": {
+        "contributors": {
+          "content": "$1",
+          "example": "<a href=\"https://github.com/rugk/…/CONTRIBUTORS.md\">all other contributors</a>"
+        }
+      }
+    },
+    "contributorsThanksLink": {
+      "message": "https://github.com/rugk/offline-qr-code/blob/master/CONTRIBUTORS.md",
+      "description": "The link to the CONTRIBUTORS file."
+    },
+    "contributorsThanksLinkText": {
+      "message": "جميع المشاركين الاخرين",
+      "description": "The link text linking to the contributors file. See contributorsThanks."
+    },
+  
+    // ARIA labels/descriptions
+    "dismissIconDescription": {
+      "message": "أغلق هذه الرسالة",
+      "description": "the aria label for the close button of the message box"
+    },
+    "ariaMessageLoading": {
+      "message": "التحميل",
+      "description": "the aria label to label the message box as an info message box"
+    },
+    "ariaMessageInfo": {
+      "message": "معلومات",
+      "description": "the aria label to label the message box as an info message box"
+    },
+    "ariaMessageSuccess": {
+      "message": "نجاح",
+      "description": "the aria label to label the message box as an success message box"
+    },
+    "ariaMessageError": {
+      "message": "خطأ",
+      "description": "the aria label to label the message box as an error message box"
+    },
+    "ariaMessageWarning": {
+      "message": "تحذير",
+      "description": "the aria label to label the message box as an warning message box"
+    }
+  }
+  
```

AFAIK there is a bracket missing:
```suggestion
}
```